### DISCUSSION TITLE
mysql host not foud error: front end  52.91.97.200  mysql 54.235.57.78-  172.31.85.251  -    updated route 53

### DISCUSSION THREAD

**Discussion by Nazrina412** [learndevopsonline/learndevopsonline#1931] [Q&A]
![image](https://github.com/user-attachments/assets/2978bfb0-7d87-4407-8af8-45e72116e69d)

**Comment by chaithanya1812** [comment]
Hi @Nazrina412 , In lab session we discussed , focus on manual steps strongly, then any kind of automation will come handy. 
![image](https://github.com/user-attachments/assets/4844a520-a19f-40f7-89ae-1701d01d8207)
